client/my-sites/checkout/composite-checkout/test/wp-checkout-order-summary.tsx

```diff
@@ -0,0 +1,168 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import config from '@automattic/calypso-config';
+import {
+	PRODUCT_JETPACK_BACKUP_T1_MONTHLY,
+	PRODUCT_JETPACK_BACKUP_T1_YEARLY,
+	PRODUCT_JETPACK_SCAN,
+} from '@automattic/calypso-products';
+import { checkoutTheme } from '@automattic/composite-checkout';
+import { ShoppingCartProvider, createShoppingCartManagerClient } from '@automattic/shopping-cart';
+import { ThemeProvider } from '@emotion/react';
+import { render, screen, waitFor } from '@testing-library/react';
+import React from 'react';
+import { Provider as ReduxProvider } from 'react-redux';
+import WPCheckoutOrderSummary from '../components/wp-checkout-order-summary';
+import {
+	mockSetCartEndpointWith,
+	mockGetCartEndpointWith,
+	createTestReduxStore,
+	getBasicCart,
+	convertProductSlugToResponseProduct,
+} from './util';
+import type { ResponseCart } from '@automattic/shopping-cart';
+import type { FC } from 'react';
+
+const mockConfig = config as unknown as { isEnabled: jest.Mock };
+jest.mock( '@automattic/calypso-config', () => {
+	const mock = () => '';
+	mock.isEnabled = jest.fn();
+	return mock;
+} );
+
+const jetpackBackupFeatureList = [
+	'Real-time cloud backups',
+	'10GB of backup storage',
+	'30-day archive & activity log',
+	'One-click restores',
+	'Priority support',
+];
```

Comment:
Nice work on adding some meaningful tests!

One minor suggestion for future component tests: it would be nice to pull the feature strings used here from the same place as the component being tested, so that tests don't need to be updated here just because a feature description changes in the component.